CMakeLists.txt

```diff
@@ -162,3 +164,8 @@ endif()
 
 lug_set_option(LUGBENCH_API_URI ${DEFAULT_LUGBENCH_API_URI} STRING "URI of the Lugbench API")
 add_definitions(/DLUGBENCH_API_URI="${LUGBENCH_API_URI}")
+
+#add api VERSION
+set(DEFAULT_LUGBENCH_API_VERSION "api/v1")
+lug_set_option(LUGBENCH_API_VERSION ${DEFAULT_LUGBENCH_API_VERSION} STRING "Version of the Lugbench API")
```

Comment:
Isn't it better to just store `v1` in this variable? Because the name is `LUGBENCH_API_VERSION` and not `LUGBENCH_API_VERSION_URI` and we will always use the URI `api/{version}`.